quesma/quesma/schema_transformer.go

```diff
@@ -152,63 +152,125 @@ func (s *SchemaCheckPass) applyIpTransformations(indexSchema schema.Schema, quer
 	return query, nil
 }
 
-func (s *SchemaCheckPass) applyGeoTransformations(currentSchema schema.Schema, query *model.Query) (*model.Query, error) {
-	fromTable := query.TableName
+func (s *SchemaCheckPass) applyGeoTransformations(schemaInstance schema.Schema, query *model.Query) (*model.Query, error) {
+
+	replace := make(map[string]model.Expr)
+
+	for _, field := range schemaInstance.Fields {
+		if field.Type.Name == schema.QuesmaTypePoint.Name {
+
+			lon := model.NewColumnRef(field.InternalPropertyName.AsString() + "::lon")
```

Comment:
This might be ok for now, however will change after introducing consistent field encoding